presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java

```diff
@@ -476,6 +478,23 @@ private void printWindowOperatorStats(int indent, WindowOperatorStats stats)
         output.append('\n');
     }
 
+    private static String formatDoubleAsCpuCost(double value)
+    {
+        return formatDoubleAsDataSize(value).replaceAll("B$", "");
+    }
+
+    private static String formatDoubleAsDataSize(double value)
+    {
+        if (!isFinite(value)) {
+            return Double.toString(value);
+        }
+        else if (isNaN(value)) {
+            return "?";
+        }
+
+        return DataSize.succinctDataSize(value, BYTE).toString();
```

Comment:
- static import `succinctDataSize`
- you can also use `succinctBytes((long) value)`